src/main/resources/org/springframework/integration/splunk/config/xml/spring-integration-splunk-1.2.xsd

```diff
@@ -0,0 +1,499 @@
+			</xsd:attribute>
+			<xsd:attribute name="owner" type="xsd:string">
+				<xsd:annotation>
+					<xsd:documentation>
+						Owner of the saved search.
+					</xsd:documentation>
+				</xsd:annotation>
+			</xsd:attribute>
+			<xsd:attribute name="app" type="xsd:string">
+				<xsd:annotation>
+					<xsd:documentation>
+						App of the saved search.
+					</xsd:documentation>
+				</xsd:annotation>
+			</xsd:attribute>
+		</xsd:complexType>
+	</xsd:element>
+
+	<xsd:complexType name="indexWriterType">
+		<xsd:attribute name="index" type="xsd:string" use="optional">
+				<xsd:annotation>
+					<xsd:documentation>
+						Index to write to.
+					</xsd:documentation>
+			</xsd:annotation>
+		</xsd:attribute>
+	</xsd:complexType>
+
+	<xsd:complexType name="tcpWriterType">
+		<xsd:attribute name="port" type="xsd:string" use="required">
+				<xsd:annotation>
+					<xsd:documentation>
+						The port corresponding to a tcp Input
+					</xsd:documentation>
+			</xsd:annotation>
+		</xsd:attribute>
+	</xsd:complexType>
+	
+	<xsd:complexType name="hecWriterType">
+		<xsd:attribute name="host" type="xsd:string" use="required">
+				<xsd:annotation>
+					<xsd:documentation>
+						The HEC host (Splunk instance address or Load Balancer address)
+					</xsd:documentation>
+			</xsd:annotation>
+		</xsd:attribute>
+		<xsd:attribute name="port" type="xsd:integer" use="required">
+				<xsd:annotation>
+					<xsd:documentation>
+						The HEC port you configured in Splunk
+					</xsd:documentation>
+			</xsd:annotation>
+		</xsd:attribute>
+		<xsd:attribute name="token" type="xsd:string" use="required">
+				<xsd:annotation>
+					<xsd:documentation>
+						The HEC token you configured in Splunk
+					</xsd:documentation>
+			</xsd:annotation>
+		</xsd:attribute>
+		<xsd:attribute name="https" type="xsd:boolean" use="optional">
+				<xsd:annotation>
+					<xsd:documentation>
+						Whether or not to use HTTPS , defaults to false
+					</xsd:documentation>
+			</xsd:annotation>
+		</xsd:attribute>
+		<xsd:attribute name="poolsize" type="xsd:integer" use="optional">
+				<xsd:annotation>
+					<xsd:documentation>
+						The HTTP connection poolsize
+					</xsd:documentation>
+			</xsd:annotation>
+		</xsd:attribute>
+		<xsd:attribute name="index" type="xsd:string" use="optional">
+				<xsd:annotation>
+					<xsd:documentation>
+						The index of the event in Splunk
+					</xsd:documentation>
+			</xsd:annotation>
+		</xsd:attribute>
+		<xsd:attribute name="source" type="xsd:string" use="optional">
+				<xsd:annotation>
+					<xsd:documentation>
+						The source of the event in Splunk
+					</xsd:documentation>
+			</xsd:annotation>
+		</xsd:attribute>
+		<xsd:attribute name="sourcetype" type="xsd:string" use="optional">
+				<xsd:annotation>
+					<xsd:documentation>
+						The sourcetype of the event in Splunk
+					</xsd:documentation>
+			</xsd:annotation>
+		</xsd:attribute>
+		<xsd:attribute name="batchMode" type="xsd:boolean" use="optional">
```

Comment:
No, it really should be in the dash style, not camel.
The `BeanDefinition` processor can transform them to the camel bean properties automatically.